src/DonationSummary/resources/js/summary.js

```diff
@@ -205,12 +205,13 @@ window.GiveDonationSummary = {
      *
      * @param {string/number} amount
      * @param {jQuery} $form
+     * @param {boolean} normalize
      */
     format_amount: function (amount, $form) {
         // Normalize amounts to JS number format
```

Comment:
@glaubersilva Can you use `Give.fn.unFormatCurrency` and `Give.fn.formatCurrency`. These functions encapsulate current formatting setting logic for a donation form and provide accurate results with add-ons.

Reply:
@glaubersilva keep in mind that currencies can have ` ` [space], `,`, `.` as thousand separators which can cause unexpected results.

Reply:
fixed: https://github.com/impress-org/givewp/commit/6c17283816f75b0f160591b112c7ac8fd9212450